docs/examples/workflows-examples.md

```diff
@@ -9,5 +9,102 @@ the [DAG Diamond example](workflows/upstream/dag_diamond.md), to demonstrate how
 These examples are generated and compared in Hera's CI/CD pipeline to ensure that all the examples are possible to
 recreate in Hera.
 
+> If you'd like to contribute missing examples, please see the table below and follow the
+> [Contributing Guide](/CONTRIBUTING.md)!
+
 The "Hera" collection shows off features and idiomatic usage of Hera such as the
 [Coinflip example](workflows/coinflip.md) using the script decorator.
+
+## List of missing examples
```

Comment:
@elliotgunton for my own clarity, these were manually copy/pasted here, right?

Reply:
Yep, but that's a good idea to automate alongside `make regenerate-test-data` 👀

Reply:
@flaviuvadan done! List is now sorted and autogenerated at the end of `make regenerate-test-data`